tests/s25Main/campaign/testCampaignLuaFile.cpp

```diff
@@ -1,16 +1,27 @@
-// Copyright (C) 2005 - 2021 Settlers Freaks (sf-team at siedler25.org)
+// Copyright (C) 2005 - 2023 Settlers Freaks (sf-team at siedler25.org)
 //
 // SPDX-License-Identifier: GPL-2.0-or-later
 
 #include "lua/CampaignDataLoader.h"
 #include "gameData/CampaignDescription.h"
 #include "rttr/test/LocaleResetter.hpp"
+#include "rttr/test/LogAccessor.hpp"
 #include "rttr/test/TmpFolder.hpp"
 #include <s25util/utf8.h>
 #include <boost/filesystem.hpp>
 #include <boost/nowide/fstream.hpp>
 #include <boost/test/unit_test.hpp>
 
+/// Require that the log contains "content"
+#define REQUIRE_LOG_CONTAINS(content)                                                                                  \
```

Comment:
Don't we have the macro already defined somewhere?

Reply:
We have only a macro defined, which checks the first line of the log and i have to check for content of multiple lines.